flixel/input/gamepad/OUYAButtonID.hx

```diff
@@ -16,8 +16,8 @@ class OUYAButtonID
 	public static inline var A:Int = 1;
 	public static inline var LB:Int = 6;
 	public static inline var RB:Int = 7;
-	public static inline var LEFT_ANALOG:Int = 10;
-	public static inline var RIGHT_ANALOG:Int = 11;
+	public static inline var LEFT_STICK_BTN:Int = 10;
```

Comment:
I'm just not a fan of the abbreviation, hadn't even thought of the click vs button thing. Is it correct to say that these are buttons / is there some official name for it? They certainly aren't "buttons" in the traditional sense.